src/containers/pages/SMC/Map/index.tsx

```diff
@@ -0,0 +1,468 @@
+import {
+  BUSINESS_STATUS,
+  CIRCLE_PAINT_COLOR_CATEGORICAL_TYPE,
+  HOME_URL,
+  REPORT_SMC_PLAN_URL,
+  SMC_REPORT_STRUCTURES,
+} from '../../../../constants';
+import { displayError } from '../../../../helpers/errors';
+import { RouteParams } from '../../../../helpers/utils';
+import supersetFetch from '../../../../services/superset';
+import GenericJurisdictionsReducer, {
+  fetchGenericJurisdictions,
+  GenericJurisdiction,
+  getGenericJurisdictionByJurisdictionId,
+  reducerName as GenericJurisdictionsReducerName,
+} from '../../../../store/ducks/generic/jurisdictions';
+import SMCPlansReducer, {
+  fetchSMCPlans,
+  getSMCPlanById,
+  reducerName as SMCPlansReducerName,
+  SMCPLANType,
+} from '../../../../store/ducks/generic/SMCPlans';
+import genericStructuresReducer, {
+  fetchGenericStructures,
+  GenericStructure,
+  getGenericStructures,
+  reducerName as genericStructuresReducerName,
+  StructureFeatureCollection,
+} from '../../../../store/ducks/generic/structures';
+import jurisdictionReducer, {
+  fetchJurisdictions,
+  getJurisdictionById,
+  Jurisdiction,
+  reducerName as jurisdictionReducerName,
+} from '../../../../store/ducks/jurisdictions';
+import {
+  buildGsLiteLayers,
+  buildJurisdictionLayers,
+  CircleColor,
+  PolygonColor,
+} from '../../FocusInvestigation/map/active/helpers/utils';
+import {
+  getIndicatorRows,
+  getJurisdictionBreadcrumbs,
+  mapOnClickHandler,
+} from '../../IRS/Map/helpers';
+import { defaultIndicatorStop, SMCIndicatorRows, SMCIndicatorStops } from './helpers';
+import './style.css';
+
+/** register the reducers */
+reducerRegistry.register(SMCPlansReducerName, SMCPlansReducer);
+reducerRegistry.register(jurisdictionReducerName, jurisdictionReducer);
+reducerRegistry.register(GenericJurisdictionsReducerName, GenericJurisdictionsReducer);
+reducerRegistry.register(genericStructuresReducerName, genericStructuresReducer);
+
+const slices = SUPERSET_SMC_REPORTING_JURISDICTIONS_DATA_SLICES.split(',');
+const focusAreaSlice = slices.pop();
+
+/** interface for SMCReportingMap */
+interface SMCReportingMapProps {
+  fetchFocusAreas: typeof fetchGenericJurisdictions;
+  fetchJurisdictionsAction: typeof fetchJurisdictions;
+  fetchPlans: typeof fetchSMCPlans;
+  fetchStructures: typeof fetchGenericStructures;
+  focusArea: GenericJurisdiction | null;
+  jurisdiction: Jurisdiction | null;
+  plan: SMCPLANType | null;
+  service: typeof supersetFetch;
+  structures: StructureFeatureCollection | null;
+}
+
+/** SMCReportingMap default props */
+const defaultProps: SMCReportingMapProps = {
+  fetchFocusAreas: fetchGenericJurisdictions,
+  fetchJurisdictionsAction: fetchJurisdictions,
+  fetchPlans: fetchSMCPlans,
+  fetchStructures: fetchGenericStructures,
+  focusArea: null,
+  jurisdiction: null,
+  plan: null,
+  service: supersetFetch,
+  structures: null,
+};
+
+/** The SMC Reporting Map component */
+const SMCReportingMap = (props: SMCReportingMapProps & RouteComponentProps<RouteParams>) => {
```

Comment:
Can you confirm that this will still load even if Superset queries fail?

Reply:
@ciremusyoka ^^

Reply:
@moshthepitt addressed by [this commit](https://github.com/onaio/reveal-frontend/pull/1234/commits/2026844d2ccd16682b6a00ffb433a5f36ed7fdf0)